hindi/Ramayana/ramayana.md

```diff
@@ -0,0 +1,3 @@
+Ramayana is one of the two major Sanskrit epics of ancient India, the other being the [Mahabharat](../Mahabharat/mahabharat.md). Along with the Mahabharat, it forms the Hindu Itihasa.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.